lib/routing.py

```diff
@@ -63,7 +69,11 @@ def route_for(self, path):
         if path.startswith(self.base_url):
             path = path.split(self.base_url, 1)[1]
 
-        for view_fun, rules in self._rules.iteritems():
```

Comment:
Does this iterate over the same items as the one in `_dispatch`? If yes, should I also just use `items()`?